src/Data/Parameterized/Peano.hs

```diff
@@ -411,5 +416,51 @@ peanoLength [] = Some zeroP
 peanoLength (_:xs) = case peanoLength xs of
   Some n -> Some (succP n)
 
+
+------------------------------------------------------------------------
+-- Some properties about Peano numbers
+
+plusCtxSizeAxiom :: forall t1 t2 f.
```

Comment:
Can you add some quick haddocks for these since they are exported?

Reply:
Done.